core/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -122,6 +123,22 @@ message AsyncCacheRequest {
 // Response for an async cache request
 message AsyncCacheResponse {}
 
+// Request for caching a block synchronously
+// next available id: 6
+message CacheRequest {
+  optional int64 block_id = 1;
+  // TODO(calvin): source host and port should be replace with WorkerNetAddress
+  optional string source_host = 2;
+  optional int32 source_port = 3;
+  optional alluxio.proto.dataserver.OpenUfsBlockOptions open_ufs_block_options = 4;
+  optional int64 length = 5;
+}
+
+// Response for an cache request
+message CacheResponse {
+  optional bool success = 1;
+}
+
```

Comment:
This new class is an exact copy of `AsyncCacheRequest`. We can rather rename the one above as `CacheRequest`, and provide async/sync distinction in `BlockWorker` method[s] alone.

Reply:
I'd also put sync/async as an additional flag into new unified `CacheRequest`. So it would be easier to manage sync/async distinction within `CacheRequestManager`.